ner.py

```diff
@@ -0,0 +1,46 @@
+from transformers import pipeline, AutoTokenizer
+
+# Load the local model and set up the pipeline
+model_name = "Gladiator/microsoft-deberta-v3-large_ner_conll2003"
+pipe = pipeline("token-classification", model=model_name)
+tokenizer = AutoTokenizer.from_pretrained(model_name)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The model and tokenizer are being loaded outside of a function or class, which means they will be loaded every time this script is imported. This could lead to performance issues if the script is imported multiple times. It would be better to load these within a function or class that can manage their lifecycle more effectively.
```diff
- model_name = "Gladiator/microsoft-deberta-v3-large_ner_conll2003"
- pipe = pipeline("token-classification", model=model_name)
- tokenizer = AutoTokenizer.from_pretrained(model_name)

+ def setup_ner_pipeline():
+     model_name = "Gladiator/microsoft-deberta-v3-large_ner_conll2003"
+     pipe = pipeline("token-classification", model=model_name)
+     tokenizer = AutoTokenizer.from_pretrained(model_name)
+     return pipe, tokenizer
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->